dat/missions/shark/sh07_pirates.lua

```diff
@@ -268,9 +274,13 @@ end
 function generic_dead()
    --Are there still other pirates to kill ?
    if mem.gawdead == true and mem.kerdead1 == true and mem.kerdead2 == true and mem.goddead == true then
-      tk.msg(_("Mission accomplished"), fmt.f(_("You have killed the four pirates. Now to return to {sys} and collect your payment..."), {sys=paysys}))
+      tk.msg(_("Mission accomplished"), fmt.f(_([["You have killed the four pirates. Now to return to {sys} and collect your payment..."]]), {sys=paysys}))
```

Comment:
This will have quotes around the message text. I assume that's not what you want? In that case you should use either only `"` or `[[`.

Reply:
Ah, so that's what that does... apologies - will fix momentarily